main/main_views/authentication.py

```diff
@@ -1,147 +0,0 @@
-import djoser
```

Comment:
This entire file was redundant of djoser. We only need to override one of the views to point to the ShopUser model. That overriding is done already in `main/main_views/user_account.py`